about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [optional] menu works well .please make the mobile menu full height so it can serve its purpose of navigating the page
![Screenshot (1296)](https://github.com/Surafels/Capstone_Project/assets/65298048/39e643dc-3eb3-47dc-9b03-ea305ed68f0f)